packages/graphql-persisted/package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@types/koa-bodyparser": "*",
     "@types/koa-compose": "*",
+    "@types/koa": "^2.0.50",
```

Comment:
This library uses the `Context` from `koa` and would require the consumer to have `koa` installed in their project. There were a number of packages that met this criteria, and I use the following method to resolve them: 1) Add the packages the host is required to have in the `peerDependencies` with a pretty loose version range, and 2) add the types packages used to the dependencies.

Why? I added the peerDependency because it is needed (yet unused) in this library. You can't really use a `koa` middleware as intended without `koa`, but you also don't need `koa` to write them. They are, more or less, just functions. I added a loose version range to reduce the chance 2 versions of the package will be installed, in this case `koa`, as a result of this library.

I added the types in dependencies because there is a middleware, `persistedOperationAssociationMiddleware`, that uses the `Context` type in one of its arguments and that function is being exported. Even though the use of the type is tangental, we are not exporting the type itself or expecting consumers will use it, they will use the function that uses the type. For this reason, it needed to be in dependencies.

Reply:
Good rationale. Is this version range too restrictive, though?